docs/concepts/computing.rst

```diff
@@ -91,7 +91,7 @@ Interactive compute session
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Interactive compute sessions are used to run various interactive applications and development tools,
-such as Jupyter Notebooks, web-based terminals, and etc.
+such as Jupyter Notebooks, web-based terminals, etc.
```

Comment:
When writing multiple numbers in English, I think it is common to write and at the end.
The comma before and may or may not be indicated, but I have seen more cases where it is not present.

Reply:
I thought "and etc." was redundant, so I corrected it. However, I have seen many documents using "and etc" also. If you think my correction was unnecessary, I'll revert it back.

Reply:
I fixed it!

Reply:
It seems better to follow @YEONFEEL96 opinion on this!